### ISSUE TITLE
SwiftLint not work with platform=linux/amd64

### ISSUE THREAD

**Issue by gulivero1773** [realm/SwiftLint#5514]
I download binary file of latest version. 
If I use `FROM --platform=linux/amd64 swift:5.9-jammy` in my CI/CD
It show to me - error: Tool ‘swiftlint’ is not supported on the target platform

**Comment by mikaelbartlett** [comment]
I wonder if it's possible to have a [SwiftLintBinary-macos.artifactbundle.zip](https://github.com/realm/SwiftLint/releases/download/0.57.0/SwiftLintBinary-macos.artifactbundle.zip) for Linux?

Having the same issue and using SPM with the build plugin. If possible to create a binary target it would be quite easy to support?

https://github.com/SimplyDanny/SwiftLintPlugins

**Comment by SimplyDanny** [comment]
As for my knowledge, SPM doesn't support binary targets on Linux.

A pre-compiled executable for arm64 on Linux should be possible though.

**Comment by mikaelbartlett** [comment]
Sorry know very little about these bundles.... 

@SimplyDanny I looked at the following... Maybe these are not the same thing? I've not looked into it much and sorry to waste your time if this is unrelated.
https://www.polpiella.dev/binary-targets-in-modern-swift-packages

**Comment by SimplyDanny** [comment]
You're right. This seems to be possible. We could either have a single `artifactbundle` or two separate ones for macOS and Linux. What's left is the build for the Linux arm64 target.
